packages/hapi/src/index.ts

```diff
@@ -1 +1,4 @@
 export { HapiAdapter } from './HapiAdapter';
+export { BullAdapter } from '@bull-board/api/bullAdapter';
+export { BullMQAdapter } from '@bull-board/api/bullMQAdapter';
+export { ExpressAdapter } from './ExpressAdapter';
```

Comment:
this is wrong adapter :]

Reply:
sorry about that 😅 I simultaneously answered your comment on the issue and sent in the PR fixed now